src/messaging.js

```diff
@@ -41,8 +41,8 @@ export default class Messaging {
 
   _handleLicensingUpdate(message) {
     if (message && message.data && message.data.is_authorized !== null && message.data.user_friendly_status) {
-      console.log(`Authorization status changed - ${message.data.user_friendly_status}`);
-      this.logger.evt(message.data.user_friendly_status ? 'Authorized' : 'Unauthorized');
+      console.log(`Authorization status updated - ${message.data.user_friendly_status}`);
```

Comment:
@Rise-Vision/content  We were already meant to log both authorized and unauthorized cases but there was a slight bug. Fixed this instead of adding more logs. Thanks!